python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -161,8 +155,17 @@ def __init__(
         # The move command that is currently running, or None.
         self.move_command = None
 
-        # Task for the current move, if one is being commanded.
-        self.move_task = salobj.make_done_future()
+        # Task for the current move. To cancel safely::
+        #
+        #     async with self.write_lock:
+        #        self.compensation_loop_task.cancel()
+        self.move_task = make_done_future()
+
+        # Compensation loop task. To cancel safely::
+        #
+        #     async with self.write_lock:
+        #        self.compensation_loop_task.cancel()
+        self.compensation_loop_task = make_done_future()
```

Comment:
I would be much more comfortable leaving it as it is because I want to keep the documentation with the attribute. Technically this should probably be a private attribute (add an underscore to the name). I omitted the underscore because this class is not designed to be inherited from, and I felt it enhanced readability a bit.